rts/System/StringUtil.h

```diff
@@ -191,6 +192,17 @@ static inline void EnsureEndsWith(std::string* str, const char* postfix)
 	}
 }
 
+static inline std::vector<std::string_view> StringSplit(std::string_view str, char sep) {
```

Comment:
Since we're C++20 perhaps this is obsoleted by `std::ranges::views::split`? Or even `lazy_split` for the use case above since we're only looking at one substring at a time.

Reply:
fancy, my brain is still at C++17, switched to ranges.